webpage/_data/navigation.yml

```diff
@@ -13,3 +13,9 @@ site_nav:
     children:
       - title: "Installing the IOOS conda environment"
         url: /other_resources
+      - title: "Australian ocean data network portal user guide"
```

Comment:
I was talking to folks here further and I think we should title the IMOS link as this " Opening netCDF files - hints from AODN"